adam-python/bdgenomics/adam/rdd.py

```diff
@@ -43,6 +43,42 @@ def __init__(self, jvmRdd, sc):
         self.sc = sc
 
 
+    def cache(self):
+        """
+        Caches underlying RDD in memory.
+
+        :return: Returns a new, cached RDD.
+        """
+
+        return self._replaceRdd(self._jvmRdd.cache())
+
+
+    def persist(self, sl):
+        """
+        Persists underlying RDD in memory or disk.
+
+        :param sl new StorageLevel
+        :return: Returns a new, persisted RDD.
+        """
+
+        jsl = self.sc._jvm.org.apache.spark.api.java.StorageLevels.create(sl.useDisk, \
```

Comment:
Do not use `\` to break lines, and please indent to the opening delimiter. I.e., every `sl` should be at the same character position on the line.